### DISCUSSION TITLE
Invoice Ninja

### DISCUSSION THREAD

**Discussion by EternalWebmedia** [runtipi/runtipi-appstore#2102] [Requests]
Have you considered adding the app yourself? 

Unfortunately, I haven't familiarized myself with GIT yet and can't do this myself. I would be very happy if someone could take over this.

**Describe app**

Invoice Ninja is an invoicing application which makes sending invoices and receiving payments simple and easy.
Hosting such systems locally is an advantage for data protection.

**Links**

- Official repository: https://github.com/invoiceninja/dockerfiles
- Official docker image: https://hub.docker.com/r/invoiceninja/invoiceninja

Include links to alln other the available resources for the app. (eg: website, license)

- License: https://invoiceninja.github.io/en/legal/license/

**Additional context**

Add any other context or screenshots about the app request here.

**Checklist**
- [ x ] I have put the app name in the title of the issue
- [ x ] I have linked the repository of the app
- [ x ] I have linked the official docker image

**Comment by CiliaFred** [comment]
[#2002](https://github.com/runtipi/runtipi-appstore/pull/2002)
